code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -379,19 +393,32 @@
 	new /obj/item/reagent_containers/food/pill/fakedeath(src)
 	new /obj/item/reagent_containers/food/pill/fakedeath(src)
 
-/obj/item/storage/pill_bottle/patch_pack/ert
-	name = "ert patch pack"
-	desc = "A patch pack containing medical patches. Issued to Nanotrasen ERT medics."
+/obj/item/storage/pill_bottle/patch_pack/ert_red
+	name = "ert red patch pack"
+	desc = "A patch pack containing medical patches. Issued to Nanotrasen ERT Red level medics."
+	wrapper_color = COLOR_NT_RED
 
 /obj/item/storage/pill_bottle/patch_pack/ert/populate_contents()
 	for(var/I in 1 to 5)
 		new /obj/item/reagent_containers/food/pill/patch/perfluorodecalin(src)
 		new /obj/item/reagent_containers/food/pill/patch/silver_sulf(src)
 		new /obj/item/reagent_containers/food/pill/patch/styptic(src)
 
+/obj/item/storage/pill_bottle/patch_pack/ert_gamma
```

Comment:
/obj/item/storage/pill_bottle/patch_pack/ert_gamma should probably be renamed to
```suggestion
/obj/item/storage/pill_bottle/patch_pack/ert/gamma
```
because it allows for us to just use the same contents of `/obj/item/storage/pill_bottle/patch_pack/ert` but just change the name/description of the bottle.